README.md

```diff
@@ -1,36 +1,194 @@
-# PRNTRS
-### **PRNTRS** - A web-app, which sends a document from customer to vendor by adding print slots thereby eliminating long wait times, inefficiency and improving the safety of the customers involved.
+# 🖨 PRNTRS — An endeavour to connect, simplify and transform
 
-#### video: https://drive.google.com/file/d/1qg76SAA9u1zIOxjDCArnTbPXwHtcCQ1s/view
+<div align="center">
 
-#### medium blog: https://medium.com/newolf-society/prntrs-a-simple-and-efficient-approach-to-remote-document-printing-6170fb1cb7cc
```

Comment:
I think you should include the link to the medium article, preferably below Video Demo link under the `Motive` category